app/database/models.py

```diff
@@ -84,6 +84,7 @@ class Event(Base):
         "UserEvent", cascade="all, delete", back_populates="events",
```

Comment:
Please fix @PureDreamer and @subscorp comments, use git precommit hooks and fix the conflicts. Thanks! :)

Reply:
Thanks, done! :)